### PR TITLE
Game Speed Decrements and Space Bar Respawn

### PR DESCRIPTION
**Pull request by kevin-moore-729**
1. Game Speed decrements by 10% after all home-spaces are filled
2. If player dies and has additional life, message appears prompting player to hit spacebar to respawn, preventing respawn by entering directional command

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -394,6 +398,7 @@ function carCollision() {
            lives--;
 
            status = 'dead';
+          //  pressSpaceToContinue.style.display = 'inline-block';
```
Comment (TheErikWolfe) [inline main.js]: Let's remove this

File: main.js [hunk 2]
```diff
@@ -416,6 +421,7 @@ function waterCollision() {
         if (count == 3) {
 
           status = 'dead';
+          // pressSpaceToContinue.style.display = 'inline-block';
```
Comment (TheErikWolfe) [inline main.js]: and this

**Review by TheErikWolfe — APPROVED** [review]

**Review by BJSalvatore — APPROVED** [review]

**Review by dimitri16h — CHANGES_REQUESTED** [review]
Fix Christians comment on the inline comments, after that it all looks good

File: main.js (hunk 2 above)
Comment (kevin-moore-729) [reply main.js]: pushed up to remove those

**Pull request merged**